### PR TITLE
Release v4.2.5

### PR DESCRIPTION
**Pull request by teddytpc1**
This PR is related to #16

### CODE REVIEW

**Review by vcerenu — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
 
 ## Prepare release
 
-**Clone repository and checkout to branch 4.1**
+**Clone repository and checkout to branch v4.2.5**
 ```
 git clone https://github.com/wazuh/wazuh-bosh
 cd wazuh-bosh
-git checkout 4.1
+git checkout v4.2.5
```
Comment (myu1d157h0u54nd) [inline README.md:9]: Could it be that there were additional steps for large files? or did that not fit here?

File: jobs/wazuh-manager-worker/monit [hunk 2]
```diff
@@ -1,55 +1,91 @@
-check process ossec-analysisd
-  matching "ossec-analysisd"
-  start program "/var/vcap/jobs/wazuh-manager-worker/bin/wazuh_server_ctl start"
-  stop program "/var/vcap/jobs/wazuh-manager-worker/bin/wazuh_server_ctl stop"
+check process wazuh-analysisd
+  matching "wazuh-analysisd"
+  start program "/var/vcap/jobs/wazuh-manager/bin/wazuh_server_ctl start"
+  stop program "/var/vcap/jobs/wazuh-manager/bin/wazuh_server_ctl stop"
```
Comment (myu1d157h0u54nd) [inline jobs/wazuh-manager-worker/monit]: jobs/wazuh-manager-worker/monit
jobs/wazuh-manager/monit
in the first you rename the path leaving him as manager
and in the second script as well?
no changes with workers?

**Review by myu1d157h0u54nd — CHANGES_REQUESTED** [review]
Review the comments please!

**Review by okynos — APPROVED** [review]
LGTM.

File: jobs/wazuh-manager-worker/monit (hunk 2 above)
Comment (teddytpc1) [reply jobs/wazuh-manager-worker/monit]: Good catch @lastranikos. This is corrected. Thanks.

File: README.md (hunk 1 above)
Comment (teddytpc1) [reply README.md:9]: The large files are now stored in S3 instead of Github. So, those steps were removed.

**Review by vcerenu — APPROVED** [review]
LGTM

File: jobs/wazuh-manager-worker/monit (hunk 2 above)
Comment (myu1d157h0u54nd) [reply jobs/wazuh-manager-worker/monit]: Thx!

**Review by myu1d157h0u54nd — APPROVED** [review]
LGTM

**Pull request merged**